README.md

```diff
@@ -104,8 +107,21 @@ Transformers4Rec library is pre-installed in the NVIDIA Merlin Docker containers
 
 Refer to the [Merlin Containers](https://nvidia-merlin.github.io/Merlin/main/containers.html) documentation page for information about the Merlin container names, URLs to the container images on the NVIDIA GPU Cloud catalog, and key Merlin components.
 
-### Feedback and Support
+## Notebook Examples and Tutorials
+
+The [End-to-end pipeline with NVIDIA Merlin](https://nvidia-merlin.github.io/Transformers4Rec/main/pipeline.html) page
+shows how to use Transformers4Rec and other Merlin libraries like NVTabular to build a complete recommender system.
 
+We have several [example](./examples) notebooks to help you build a recommender system or integrate Transformers4Rec into your system:
+
+- A getting started example that includes training a session-based model with an XLNET transformer architecture.
+- An end-to-end example that trains a model and takes the next step to serve inference with Triton Inference Server.
+- Another end-to-end example that trains and evaluates a session-based model on RNN and also serves inference with Triton Inference Server.
+- A notebook and scripts that reproduce the experiments presented in a paper for RecSys 2021.
```

Comment:
Maybe adding links to the individual example notebooks as you list them here would be useful for the user.

Reply:
I intentionally didn't add links to the notebooks because we have an intermediate page at https://nvidia-merlin.github.io/Transformers4Rec/main/examples/index.html.  Unless we choose to ditch the intermediate page, my inclination is to guide readers past it.